clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -309,3 +271,40 @@ func apiCallToPerfData(apicalls *apiResponsiveness) *measurementutil.PerfData {
 	}
 	return perfData
 }
+
+func (m *apiCallMetrics) sorted() []*apiCallMetric {
+	all := make([]*apiCallMetric, 0)
+	for _, v := range m.metrics {
+		all = append(all, v)
+	}
+	sort.Slice(all, func(i, j int) bool {
+		return all[i].Latency.Perc99 < all[j].Latency.Perc99
```

Comment:
This is sorting it in increasing order. Which isn't what we want, because in line 188 you now don't print the top metrics..

Am I missing something?

Reply:
Good catch. Fix with tests in https://github.com/kubernetes/perf-tests/pull/1229.